cmd/vms_test.go

```diff
@@ -35,6 +35,7 @@ var _ = Describe("VMsCmd", func() {
 
 		BeforeEach(func() {
 			opts = VMsOpts{}
+			opts.ParallelOpt = 5
```

Comment:
Is it possible to have additional test coverage for this new behavior? We see the integration test in the bosh director which looks good, but I wonder if we can cover at a unit level here.

Would it be practical to mock the `VMInfos` calls such that a first call is long, but second call is short? Then there could be some assertions around the second call being finished before the first to ensure the parallel behavior is working as expected. Just one idea - not sure if you have other ideas about it.